governance/program/src/state/governance.rs

```diff
@@ -225,10 +223,7 @@ impl GovernanceV2 {
         let vote_threshold = if realm_data.community_mint == *vote_governing_token_mint {
             match vote_kind {
                 VoteKind::Electorate => &self.config.community_vote_threshold,
-                VoteKind::Veto => {
-                    // Community Veto vote is not supported in current version
-                    return Err(GovernanceError::GoverningTokenMintNotAllowedToVote.into());
-                }
+                VoteKind::Veto => &self.config.community_veto_vote_threshold,
```

Comment:
The default for existing DAOs `Community Veto` is `Disabled` . It's set in the `Governance` account  deserialisation code:

```
 // For legacy accounts set the community Veto threshold to Disabled
 governance_data.config.community_veto_vote_threshold = VoteThreshold::Disabled;
```
However I realised I missed the assertion for `community_veto_vote_threshold` and added it in this commit fdb0da764e44108530f8b895510b6f4a4f91bfea

Reply:
I'm probably missing something, but that's only triggered based on `council_vote_threshold`, not on `community_veto_vote_threshold`. Will that be ok?

Reply:
Going off your other comment, this should be fine. In the future, to make this clearer, it might be clearer to add a version identifier to these structs and go off of that, which will also make the logic clearer

Reply:
Absolutely, it would be much better to have a `version` field. I've managed to make all the changes for `V3` backward compatible and hence there was no need to introduce new account type `GovernanceV3`. 

The versioned account types are killing our `RPCs` with too many `getProgramAccounts` and I want to introduce a `version` field instead but because there were no breaking changes this time I went with lesser evil and put that logic in the account upgrade code.